src/repl.ts

```diff
@@ -230,6 +231,27 @@ function enterPasteMode() {
   })
 }
 
+function loadFile(filename: string) {
+  try {
+    let filePath = path.resolve(filename)
+    let fileContents: string = fs.readFileSync(filePath, 'utf8')
```

Comment:
Please remove the annotation since fs.readFileSync's return type is clearly string